actionpack/lib/action_dispatch.rb

```diff
@@ -95,6 +95,7 @@ module Session
     autoload :TestProcess
     autoload :TestRequest
     autoload :TestResponse
+    autoload :AssertionResponse
```

Comment:
Instead of autoloading I think it is better to require the file where it is needed.